_docs_integrate/63-attribute-values.md

```diff
@@ -449,6 +449,18 @@ The officially registered pseudonym of a person.
 | `@type` | `"Pseudonym"` |    ✓     |                  |
 | `value` | `string`      |    ✓     | max. length: 100 |
 
+## SchematizedXML
+
+SchematizedXML can be used to exchange files in XML format. The exchange of XML files is also possible via [`FileReference`](#filereference), but SchematizedXML has the advantage that it is possible to validate the XML and display the attributes in the wallet.
+
+**Properties**
+
+| Name        | Type               | Required | Validation                                                 |
+| ----------- | ------------------ | :------: | ---------------------------------------------------------- |
+| `@type`     | `"SchematizedXML"` |    ✓     |                                                            |
+| `value`     | `string`           |    ✓     | max. length: 50000 <br>must be a valid XML                 |
```

Comment:
isn't it an `XML document` instead of only XML?

Reply:
plain xml string, not a file.

Reply:
xml is a format, I dont think there is `an xml`, right?

Reply:
wouldn't the same be true for url format?

Reply:
IMO we talk about URLs in another way than "XMLs". But maybe it's just my gusto.